Python/Product/Analysis/Interpreter/Ast/AstScrapedPythonModule.cs

```diff
@@ -25,19 +27,27 @@
 
 namespace Microsoft.PythonTools.Interpreter.Ast {
     class AstScrapedPythonModule : IPythonModule {
-        private readonly string _filePath;
+        private readonly string _name, _filePath;
+        private readonly bool _isBuiltins;
```

Comment:
Yeah, I was starting to think that when adding the tests. Probably worth doing now.